nameable.rb

```diff
@@ -0,0 +1,5 @@
+module Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
```

Comment:
```ruby
class Nameable
  def correct_name
    raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
  end
end
```

- [ ] As suggested by the previous reviewer that this should not be a module but it should be a class to follow the project requirements :+1: it will be like below :arrow_down:
- [ ] It would be great if you and the next reviewer check your app after making these changes :+1: to make sure everything is working as expected :blush: